### PR TITLE
getDownloadObject: use openpolicyagent.org redirect, allow for "edge" version

### PR DESCRIPTION
**Pull request by srenatus**
I've needed an extra if branch in `getVersion()` to make this work. Before, `"latest"` was also declined as an invalid range or version.

Using the redirect, we get homogeneous URLs for all versions, latest, and edge binaries. (They're defined here: https://github.com/open-policy-agent/opa/blob/v0.35.0/docs/website/layouts/index.redirects#L35-L38)

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -67,6 +68,10 @@ async function renameBinary(
 
 async function getVersion(): Promise<string> {
   const version = core.getInput('version');
+  if (version === 'latest' || version === 'edge') {
+    return version;
+  }
```
Comment (srenatus) [inline src/index.ts:73]: Reading the code again, the warning had misled me into believing it didn't work. So this function worked with "latest" before, and would work with "edge" now; adding the block here just gets rid of the warning. WDYT?

**Review by tim775 — COMMENTED** [review]
This makes sense to me.  Do we need to update the test case for "latest" at all?  Right now we look up the expected version from https://api.github.com/repos/open-policy-agent/opa/releases/latest see (https://github.com/srenatus/setup-opa/blob/sr/add-edge-and-use-redirect/.github/workflows/test_setup_opa.yml#L40)

Also can/should we add a test case for "edge"?  I'm not sure where to look up the expected version for that.

**Comment by srenatus** [comment]
> Also can/should we add a test case for "edge"? I'm not sure where to look up the expected version for that.

Yeah just what I had in mind 🙃 gave it a go in another commit. Ah, need to fix something there wrt. patch releases.... one moment.

**Comment by srenatus** [comment]
"edge" doesn't work. I'm looking into it.

**Comment by srenatus** [comment]
💡 didn't update `dist` in my PR

**Review by tim775 — APPROVED** [review]

**Comment by tim775** [comment]
Looks good, thanks!  I'm going to add an action to make sure the dist directory has been updated.
